docs/en/how_to_guides/finetune_with_a_custom_dataset.md

```diff
@@ -0,0 +1,357 @@
+# Fine-tune with A Custom Dataset
+
+This document introduces the process for fine-tuning a custom dataset in MindCV and the implementation of fine-tuning techniques such as reading the dataset online, setting the learning rate for specific layers, freezing part of the parameters, etc. The main code is in./example/finetune.py, you can make changes to it based on this tutorial as needed.
+
+Next, we will use the FGVC-Aircraft dataset as an example to show how to fine-tune the pre-trained model mobilenet v3-small. [Fine-Grained Visual Classification of Aircraft](https://www.robots.ox.ac.uk/~vgg/data/fgvc-aircraft/) is a commonly used fine-grained image Classification benchmark dataset, which contains 10,000 aircraft images from 100 different types of aircraft (a.k.a variants), that is, 100 images for each aircraft type.
+
+First, extract the [downloaded](https://mindspore-lab.github.io/mindcv/zh/tutorials/finetune/#_3) dataset to . /data folder, the directory structure of the Aircraft dataset is:
+
+```text
+aircraft
+└── data
+    ├── images
+    │   ├── image1.jpg
+    │   ├── image2.jpg
+    │   └── ....
+    ├──	images_variant_test.txt
+    ├──	images_variant_trainval.txt
+    └── ....
+```
+
+The folder "images" contains all the 10,000 images, and the airplane types and subset names of each image are recorded in images_variant_*.txt. When this dataset is used for fine-tuning, the training set is usually set by annotation file: images_variant_trainval.txt. Hence, the training set should contain 6667 images and the test set should contain 3333 images after the dataset has been split.
+
+## Data Preprocessing
+
+### Read Custom Dataset
+
+For custom datasets, you can either organize the dataset file directory locally into a tree structure similar to ImageNet, and then use the function `create_dataset` to read the dataset (offline way), or directly read all the images into an iterable object, replacing the file splitting and the `create_dataset` steps (online way).
```

Comment:
Since the structure of downstream datasets can be diverse, to be able to handle this diversity, the following two approaches are common practice:

Offline processing: Manually reorganize data into a standard format for loading by existing interfaces. The keyword **_offline_** is reflected in the process of manually formatting the dataset.

Online processing: Building new interfaces to be able to load datasets in specific formats. The keyword **_online_** is reflected in the fact that the newly constructed interface is parsing the data structure during runtime.